packages/sdk-communication-layer/src/services/RemoteCommunication/ConnectionManager/initCommunicationLayer.ts

```diff
@@ -94,7 +94,7 @@ export function initCommunicationLayer({
     url,
     title,
     source: state.dappMetadata?.source,
-    icon: state.dappMetadata?.iconUrl,
+    icon: state.dappMetadata?.iconUrl || state.dappMetadata?.base64Icon,
```

Comment:
please name this property `icon` to keep compatibility with what's it's name already

Reply:
`base64Icon` was it's name before it got removed

Reply:
my fault, it was base64Icon :(